sdk/servicebus/Azure.Messaging.ServiceBus/tests/Receiver/ReceiverLiveTests.cs

```diff
@@ -280,7 +282,7 @@ await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitionin
 
                 ServiceBusSender sender = client.CreateSender(scope.QueueName);
                 using ServiceBusMessageBatch batch = await sender.CreateMessageBatchAsync();
-                IEnumerable<ServiceBusMessage> messages = ServiceBusTestUtilities.AddMessages(batch, messageCount).AsReadOnly<ServiceBusMessage>();
+                IEnumerable<AmqpMessage> messages = ServiceBusTestUtilities.AddMessages(batch, messageCount).AsReadOnly<AmqpMessage>();
```

Comment:
Hmm, why do we want to operate on AmqpMessage for this test?

Reply:
The batch no longer stores `ServiceBusMessage` instances, only `AmqpMessage`.   Unless you're using a special mocking batch, there are no models to return.

Reply:
Ah I see. Rather than leaking AmqpMessage throughout all the tests, I think we should just use the SendMessagesAsync overload that takes ServiceBusMessage. Alternatively, we could add a utility method that creates a batch from a set of ServiceBusMessages.